ex/high_throughput_inference/mock_app.py

```diff
@@ -126,38 +92,44 @@ def run_model(self, model: bytes | str, batch: torch.Tensor):
             output_descriptors=[],
             custom_attributes=None,
         )
-        self.measure_time("build_request")
+        self._perf_timer.measure_time("build_request")
         request_bytes = MessageHandler.serialize_request(request)
-        self.measure_time("serialize_request")
+        self._perf_timer.measure_time("serialize_request")
+        tensor_bytes = [bytes(tensor.data) for tensor in tensors]
+        # tensor_bytes = [tensor.reshape(-1).view(numpy.uint8).data for tensor in tensors]
+        self._perf_timer.measure_time("serialize_tensor")
         with self._to_worker_fli.sendh(timeout=None, stream_channel=self._to_worker_ch) as to_sendh:
             to_sendh.send_bytes(request_bytes)
-            for t in tensors:
-                to_sendh.send_bytes(t.tobytes()) #TODO NOT FAST ENOUGH!!!
+            for tb in tensor_bytes:
+                to_sendh.send_bytes(tb) #TODO NOT FAST ENOUGH!!!
                 # to_sendh.send_bytes(bytes(t.data))
-        logger.info(f"Message size: {len(request_bytes)} bytes")
 
-        self.measure_time("send")
+        self._perf_timer.measure_time("send")
         with self._from_worker_ch.recvh(timeout=None) as from_recvh:
             resp = from_recvh.recv_bytes(timeout=None)
-            self.measure_time("receive")
+            self._perf_timer.measure_time("receive_response")
             response = MessageHandler.deserialize_response(resp)
-            self.measure_time("deserialize_response")
+            self._perf_timer.measure_time("deserialize_response")
             # list of data blobs? recv depending on the len(response.result.descriptors)?
-            data_blob = from_recvh.recv_bytes(timeout=None)
+            data_blob: bytes = from_recvh.recv_bytes(timeout=None)
+            self._perf_timer.measure_time("receive_tensor")
             result = torch.from_numpy(
                 numpy.frombuffer(
                     data_blob,
                     dtype=str(response.result.descriptors[0].dataType),
                 )
             )
-            self.measure_time("deserialize_tensor")
+            self._perf_timer.measure_time("deserialize_tensor")
 
-        self.end_timings()
+        self._perf_timer.end_timings()
         return result
 
     def set_model(self, key: str, model: bytes):
         self._ddict[key] = model
 
+    def print_timings(self, to_file: bool):
```

Comment:
`to_file` is unused.

do we really need this 1-liner method?

Reply:
Nope, but I'll expose the perf timer to avoid lint errors